website/notifications/emails.py

```diff
@@ -0,0 +1,143 @@
+import datetime
+import urlparse
+from modularodm import Q
+from modularodm.exceptions import NoResultsFound
+from model import Subscription, DigestNotification
+from framework.tasks import app
+from framework.tasks.handlers import queued_task
+from website import mails, settings
+from website.util import web_url_for
+from website.models import Node, User
+from mako.lookup import Template
+
+
+def notify(uid, event, **context):
+    key = str(uid + '_' + event)
```

Comment:
This sort of formatting is an implementation detail that should be hidden. Can you move this to a tiny little helper function `to_subscription_key(uid, event)`.?